external/storm-submit-tools/src/main/java/org/apache/storm/submit/dependency/Booter.java

```diff
@@ -37,16 +37,19 @@ public static RepositorySystemSession newRepositorySystemSession(
             RepositorySystem system, String localRepoPath) {
         MavenRepositorySystemSession session = new MavenRepositorySystemSession();
 
-        // find homedir
-        String home = System.getProperty("storm.home");
-        if (home == null) {
-            home = ".";
-        }
+        File repoDir = new File(localRepoPath);
```

Comment:
This code is from Zeppelin and I just fixed the path handling bug, but I think your suggestion would be better. I'll reflect it.